src/wp-content/themes/twentyseventeen/inc/block-patterns.php

```diff
@@ -0,0 +1,199 @@
+		array(
+			'title'      => __( 'Large Heading with Button', 'twentyseventeen' ),
+			'categories' => array( 'twentyseventeen' ),
+			'content'    => '<!-- wp:heading {"level":1,"textColor":"black","style":{"typography":{"fontSize":50}}} -->
+            <h1 class="has-black-color has-text-color" style="font-size:50px">' . __( 'Attract Leads with Marketing Campaigns that Work', 'twentyseventeen' ) . '</h1>
+            <!-- /wp:heading -->
+
+            <!-- wp:buttons -->
+            <div class="wp-block-buttons"><!-- wp:button {"borderRadius":0,"className":"is-style-fill"} -->
+            <div class="wp-block-button is-style-fill"><a class="wp-block-button__link no-border-radius">' . __( 'Our Services', 'twentyseventeen' ) . '</a></div>
+            <!-- /wp:button --></div>
+            <!-- /wp:buttons -->',
+		)
+	);
+
+	register_block_pattern(
+		'twentyseventeen/images-with-text-and-link',
+		array(
+			'title'      => __( 'Images with Text and Link', 'twentyseventeen' ),
+			'categories' => array( 'twentyseventeen' ),
+			'content'    => '<!-- wp:spacer -->
+            <div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+            <!-- /wp:spacer -->
+            <!-- wp:columns -->
+            <div class="wp-block-columns"><!-- wp:column -->
+            <div class="wp-block-column">
+			<!-- wp:image {"className":"size-large"} -->
+			<figure class="wp-block-image size-large"><img src="' . get_template_directory_uri() . '/assets/images/stripes.jpg" alt="' . __( 'Black Stripes', 'twentyseventeen' ) . '"/></figure>
+			<!-- /wp:image -->
+            <!-- wp:heading {"textColor":"black","style":{"typography":{"fontSize":45}}} -->
+            <h2 class="has-black-color has-text-color" style="font-size:45px">' . __( 'Branding', 'twentyseventeen' ) . '</h2>
+            <!-- /wp:heading -->
+            <!-- wp:paragraph {"textColor":"black","style":{"typography":{"lineHeight":"1.8"}}} -->
+            <p class="has-black-color has-text-color" style="line-height:1.8">' . __( 'Communicate your purpose and goals with a beautiful logo that encapsulates your business.', 'twentyseventeen' ) . '</p>
+            <!-- /wp:paragraph -->
+            <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+            <p style="line-height:3"><a href="#"><strong>' . __( 'See Case Study', 'twentyseventeen' ) . ' →</strong></a></p>
+            <!-- /wp:paragraph --></div>
+            <!-- /wp:column -->
+            <!-- wp:column -->
+            <div class="wp-block-column"><!-- wp:spacer {"height":254} -->
+            <div style="height:254px" aria-hidden="true" class="wp-block-spacer"></div>
+            <!-- /wp:spacer -->
+			<!-- wp:image {"className":"size-large"} -->
+			<figure class="wp-block-image size-large"><img src="' . get_template_directory_uri() . '/assets/images/white-border.jpg" alt="' . __( 'White border', 'twentyseventeen' ) . '"/></figure>
```

Comment:
This URl needs to be escaped.

Reply:
@carolinan [see the discussion above](https://github.com/WordPress/wordpress-develop/pull/606#discussion_r508856895). I'm happy to escape these if they need to be escaped, but there is some confusion around it.